projects/demo/src/modules/components/link/examples/3/index.html

```diff
@@ -23,7 +24,8 @@
     iconEnd="@tui.circle-help"
     tuiLink
     type="button"
-    [pseudo]="true"
+    [style.text-decoration-line]="'underline'"
```

Comment:
Same here

Reply:
Without [style.text-decoration-line]="'underline'" this is an angular 16 bug

Reply:
fixed